lua/astrocommunity/programming-language-support/hurl-nvim/README.md

```diff
@@ -0,0 +1,12 @@
+# hurl.nvim
+
+Hurl.nvim is a Neovim plugin designed to run HTTP requests directly from `.hurl` files. Elevate your API development workflow by executing and viewing responses without leaving your editor.
+
+### Dependencies
+
+- System-wide
+  - jq
+  - hurl
+  - prettier
```

Comment:
```suggestion
- System-wide
  - jq
  - hurl
  - prettier
```
`jq` and `prettier` are not hard dependencies. At best they are optional ones. @iamralch